kolibri/plugins/coach/assets/src/views/exams-page/change-exam-visibility-modal.vue

```diff
@@ -38,10 +39,9 @@
   export default {
     name: 'changeExamVisibilityModal',
     $trs: {
-      examVisibility: 'Exam visibility',
-      shouldBeVisible: "'{ examTitle }' should be visible to:",
+      examVisibility: 'Change exam visibility',
+      shouldBeVisible: `Make '{ examTitle }' visible to entire class or specific groups`,
```

Comment:
Using backticks without string interpolation looks kind of weird. Is it critical that `examTitle` be in single quotes? You could also do the whole message in single quotes, but escape the quotes in the middle `'...\'{examTitle}\'..'`

Reply:
This is something that should be formatted by `prettier`.

In the past, I've seen it use the strategy shown in the removed line - wrapping with double quotes:

```javascript
"Make '{ examTitle }' visible to entire class or specific groups"
```

Reply:
Yeah, maybe go back to using double-quotes for the whole thing. Prettier will not correct using unnecessary backticks (certain ESLint configs would, though). But prettier will actually autoformat 'hello i said 'hello'' to "hello i said 'hello'" (i.e. this is how it prefers to handle single-quotes).